engine/common/api_helpers/utils.py

```diff
@@ -45,6 +48,41 @@ def __repr__(self):
         return "%s()" % self.__class__.__name__
 
 
+class CustomURLValidator(URLValidator):
```

Comment:
I propose to rename that to "URLValidatorWithoutTLD" or something like that.

Reply:
I tried it. However, the overridden process is not effective unless we override the [regex](https://github.com/django/django/blob/main/django/core/validators.py#LL95C3-L95C3) attribute. To override it I've to bring all other necessary variables (ip4_re, ip6_re, host_re).
I couldn't find a way to make it more concise. It would be great if you found a better way.

In terms of the class name, I will change it to what you proposed.